FreeSurfer/FreeSurferPipeline.sh

```diff
@@ -625,14 +637,16 @@ main()
 	if [ "${existing_subject}" != "TRUE" ]; then  # input volumes only necessary first time through
 		recon_all_cmd+=" -all"
 		recon_all_cmd+=" -i ${zero_threshold_T1wImage}"
-		recon_all_cmd+=" -T2 ${T2wImage}"
+		[ "${T2wImage}" != "NONE" ] && recon_all_cmd+=" -T2 ${T2wImage}"
 		recon_all_cmd+=" -emregmask ${T1wImageBrain}"
 	fi
 
 	# By default, refine pial surfaces using T2.
 	# If for some reason the -T2pial flag needs to be excluded from recon-all, this can be
 	# accomplished using --extra-reconall-arg=-noT2pial
```

Comment:
Update comments:
```
# By default, refine pial surfaces using T2 (if T2w image provided).
# If for some other reason the -T2pial flag needs to be excluded from recon-all, 
# this can be accomplished using --extra-reconall-arg=-noT2pial
```

Reply:
Updated.